tests/test_inner.py

```diff
@@ -218,18 +232,19 @@ class ExitCompat(WatchedMixIn, ExitPlugin):
 
 
 class Watcher(object):
-    def __init__(self):
+    def __init__(self, rais=False):
```

Comment:
Typo in parameter name, should this be 'raise'? Why not create a new class using the RaisesMixIn?